packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -37,6 +37,7 @@ export default class BuySellStore extends BaseStore {
     sort_by = 'rate';
     submitForm = null;
     table_type = buy_sell.BUY;
+    temp_contact_info = null;
```

Comment:
reason why we need this is to make sure the values for contact_info stay there, even after a rerender. The re rendering is caused by the market rate changing, and because the initial value of contact_info is an empty string, it will reset it back to that value. 

i had tried with useRef, but this won't update the input value when user is typing in the input field.